src/main/java/com/google/codeu/servlets/FeedServlet.java

```diff
@@ -0,0 +1,52 @@
+package com.google.codeu.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.codeu.data.Datastore;
+import com.google.codeu.data.Message;
+
+import java.io.IOException;
+import java.util.List;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Handles fetching all messages for the public feed.
```

Comment:
When I read this comment I thought this servlet was going to return JSON, but it actually fetches the messages and then forwards to a JSP for display. Maybe update the comment?

Reply:
Oh yeah, there was a lot of copy pasta going into this, my bad.